packages/next/src/export/index.ts

```diff
@@ -507,6 +507,7 @@ export async function exportAppImpl(
       : {}),
     strictNextHead: !!nextConfig.experimental.strictNextHead,
     deploymentId: nextConfig.experimental.deploymentId,
+    swrDelta: nextConfig.swrDelta,
```

Comment:
Let's keep this in experimental to start, then we can stabilize after some testing/validating

Reply:
noted, adjusted to `experimental`